packages/default-theme/components/checkout/Payment.vue

```diff
@@ -0,0 +1,529 @@
+<template>
+  <div>
+    <SfHeading
+      title="3. Payment"
+      class="sf-heading--left sf-heading--no-underline title"
+    />
+    <div class="form">
+      <SfCheckbox
+        v-model="sameAsShipping"
```

Comment:
this is checked on start but the address is not copied, but nevermind and look at main Review note, because flow here should be different :)

Reply:
I've created appropriate issue and I've put it in the refinement project.